egs/wsj/s5/utils/text_to_phones.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
```

Comment:
this script is kind of specialized so I'd rather not put it here-- please move it to one of the 'e2e' directories in steps/.